manuscript/How-secure-is-portugal.md

```diff
@@ -14,7 +14,11 @@ The question is, how secure are they? How safe are they? By 'secure', I mean how
 
 ### Our security model is based on lack of attackers
 
-Portugal is safe because there are not enough attackers targeting the current insecurities of the system. This will probably remain the case for the next couple of years. The problem is what happens after that, when the criminals improve their business models and start to focus on Portuguese assets.
+I use the internet, and I use my credit card online, even though I know I might be hacked by a criminal.
+
+I know what the underbelly looks like, so if anybody should be paranoid, it should be me. Despite this, I feel pretty safe from a cyber attack. 
```

Comment:
These two phases are not working and move away from the topic of this paragraph.

I think it is better to remove them